src/main/resources/changelog/scripts/v-1.0.0/resource_graph/tables/instance_metadata.sql

```diff
@@ -0,0 +1,12 @@
+create table if not exists instance_metadata (
+     id serial not null primary key,
```

Comment:
As noted in the resources table, I would make this 
`resource_hash bigint prmary key references resources.resource_hash`

Reply:
Also it should be partitioned using the same scheme as the resources table based on the resource_hash.

Reply:
Done.